picky-asn1-x509/src/private_key_info.rs

```diff
@@ -48,7 +55,8 @@ use std::fmt;
 #[derive(Serialize, Debug, Clone, PartialEq)]
 pub struct PrivateKeyInfo {
     pub version: u8,
-    pub private_key_algorithm: AlgorithmIdentifier,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub private_key_algorithm: Option<AlgorithmIdentifier>,
```

Comment:
Apparently this is mixing the PKCS8 format with the EC private key format. I expect this field to stay non optional.